src/components/Comments/WriteComment.js

```diff
@@ -199,7 +200,7 @@ export const styles = StyleSheet.create({
     flex: 1
   },
   btn: (isDisableSubmit) => ({
-    backgroundColor: !isDisableSubmit ? colors.bondi_blue : '#f2f2f2',
+    backgroundColor: !isDisableSubmit ? colors.bondi_blue : COLORS.concrete,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, this change improves maintainability and consistency by using a centralized theme for colors. Make sure `COLORS.concrete` is defined in your theme object and it represents the same color as the previous hardcoded value `#f2f2f2`.
```diff
-    backgroundColor: !isDisableSubmit ? colors.bondi_blue : '#f2f2f2',
+    backgroundColor: !isDisableSubmit ? colors.bondi_blue : COLORS.concrete,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->